### PR TITLE
AUX outputs

### PR DESCRIPTION
**Pull request by rusefillc**
(cherry picked from commit 211849d245e6af56267b0c566786cc8eb1cb4056)

### CODE REVIEW

File: firmware/auxout.cpp [hunk 1]
```diff
@@ -0,0 +1,65 @@
+#include "pwm.h"
+#include "lambda_conversion.h"
+
+#include "wideband_config.h"
+
+#include "hal.h"
+
+#ifdef AUXOUT_DAC_PWM_DEVICE
```
Comment (mck1117) [inline firmware/auxout.cpp:8]: shall we define this somewhere so we get ci coverage?

Comment (rusefillc) [reply firmware/auxout.cpp:8]: Very reasonable idea. As you know f103 has a few unused pins, please consider helping by enabling thisbfor f1_dave

Comment (dron0gus) [reply firmware/auxout.cpp:8]: We need f1_rev2 to be merged first. It has AUX outs.

Comment (mck1117) [reply firmware/auxout.cpp:8]: merged!

**Comment by rusefillc** [comment]
replaced by #95

**Pull request closed without merging**